app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -284,7 +298,7 @@ import TransactionStatus from 'src/components/TransactionStatus.vue';
 import LoadingSpinner from 'src/components/LoadingSpinner.vue';
 import { CLR, fetch, InitArgs, linear } from '@dgrants/dcurve';
 // --- Icons ---
-import { ArrowToprightIcon as ShareIcon } from '@fusion-icons/vue/interface';
+import { TwitterIcon as TwitterIcon } from '@fusion-icons/vue/interface';
```

Comment:
```suggestion
import { TwitterIcon } from '@fusion-icons/vue/interface';
```

Reply:
pushed right now removing in 2 files the unnessesary "as TwitterIcon"